python/helpers/lib/parser.py

```diff
@@ -2,17 +2,28 @@
 import glob
 import io
 import json
+import optparse
 import os.path
 import re
 
 import setuptools
 import pip._internal.req.req_file
 from pip._internal.download import PipSession
+from pip._internal.models.format_control import FormatControl
 from pip._internal.req.constructors import install_req_from_line
 
+JINJA_DELIMITER_IGNORE_REGEX = r"({{(.*?)}})|({%[-]?(.*?)%})|({#(.*?)#})"
+
 def parse_requirements(directory):
     # Parse the requirements.txt
     requirement_packages = []
+    parser_options = optparse.Values(
+            {
+                "skip_requirements_regex": JINJA_DELIMITER_IGNORE_REGEX,
+                "isolated_mode": False,
+                "format_control": FormatControl(),
+            }
```

Comment:
Neat! What does `isolated_mode` and `format_control` do here? Could you add a comment to explain these settings? Can definitely see myself forgetting in future 🤠